service/patch.go

```diff
@@ -83,11 +83,16 @@ func (uis *UIServer) patchPage(w http.ResponseWriter, r *http.Request) {
 	commitQueuePosition := 0
 	if projCtx.Patch.Alias == evergreen.CommitQueueAlias {
 		cq, err := commitqueue.FindOneId(project.Identifier)
+		// still display patch page if problem finding commit queue
 		if err != nil {
 			uis.LoggedError(w, r, http.StatusInternalServerError, errors.Wrap(err, "error finding commit queue"))
 		}
 		if cq != nil {
 			commitQueuePosition = cq.FindItem(projCtx.Patch.Id.Hex())
+			// if CLI commit queue, an item not on the queue may be paused
+			if commitQueuePosition < 0 && projCtx.ProjectRef.CommitQueue.PatchType != commitqueue.CLIPatchType {
+				uis.LoggedError(w, r, http.StatusNotFound, errors.New("item not on commit queue"))
```

Comment:
will this log an error when a user loads a patch testing a PR item that's already been deleted or tested and removed?
Do we need to catch this error?

Reply:
Hmm yes it'll log an error. Perhaps this error doesn't need to be caught, since we give a message in the UI (and this message would only be shown if the version isn't available).